net/mwan3/files/usr/sbin/mwan3rtmon

```diff
@@ -155,7 +155,7 @@ main()
 		IP="$IP4"
 	fi
 	mwan3_init
-	sh -c "echo \$\$; exec $IP monitor route" | {
+	sh -c "KILL -SIGSTOP $$; echo \$\$; exec $IP monitor route" | {
```

Comment:
@aaronjg Can you explain to me the problem you are fixing here?

Reply:
We must handle the case when new routes are being added while we are processing the initial routing table. I had initially solved this by suspending the ip monitor reading process shortly after it was started, but that still allowed for a race condition.

This was designed to suspend the process a bit sooner. On closer inspection, it appears it was actually suspending the monitor process, rather than the reading process, so I fixed this with 34b58f40.

This is a pretty edge case that is being handled, so if you want to keep this P/R cleaner I can drop that commit and the subsequent one from this P/R.